dbfit-java/core/src/main/java/dbfit/fixture/FunctionResultSetStatementExecution.java

```diff
@@ -1,14 +1,12 @@
-package dbfit.environment;
+package dbfit.fixture;
 
 import java.sql.*;
 
-import dbfit.fixture.StatementExecution;
-
-public class InformixFunctionStatementExecution extends StatementExecution {
+public class FunctionResultSetStatementExecution extends StatementExecution {
```

Comment:
The function return value is retried via a `ResultSet` rather than via a `RETURN_VALUE` parameter. There's probably a better name yet to be imagined.